hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveCommitLocks.java

```diff
@@ -88,13 +95,33 @@ public class TestHiveCommitLocks extends HiveTableBaseTest {
   LockResponse notAcquiredLockResponse = new LockResponse(dummyLockId, LockState.NOT_ACQUIRED);
   ShowLocksResponse emptyLocks = new ShowLocksResponse(Lists.newArrayList());
 
-  @BeforeAll
-  public static void startMetastore() throws Exception {
-    HiveMetastoreTest.startMetastore(
-        ImmutableMap.of(HiveConf.ConfVars.HIVE_TXN_TIMEOUT.varname, "1s"));
-
+  private static final String DB_NAME = "hivedb";
+  private static final String TABLE_NAME = "tbl";
+  private static final Schema schema =
+      new Schema(Types.StructType.of(required(1, "id", Types.LongType.get())).fields());
+  private static final PartitionSpec partitionSpec = builderFor(schema).identity("id").build();
+  static final TableIdentifier TABLE_IDENTIFIER = TableIdentifier.of(DB_NAME, TABLE_NAME);
+
+  @RegisterExtension
+  private static final HiveMetastoreExtension HIVE_METASTORE_EXTENSION =
+      new HiveMetastoreExtension(
+          DB_NAME, ImmutableMap.of(HiveConf.ConfVars.HIVE_TXN_TIMEOUT.varname, "1s"));
+
+  private HiveCatalog catalog;
+
+  private void startMetastore() throws Exception {
```

Comment:
this method is not starting the `metastore`

Reply:
indeed I missed this, let's rename it to ìnitCatalog()`